js/Tooltips/tooltipsFactory.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const tippy = require('tippy.js').default;
+const { hideAll } = require('tippy.js');
+const TooltipListener = require('./TooltipListener');
+
+/**
+ * Create a Tooltip listener instance
+ * @param {jQuery} $html - jQuery wrapper of the html node
+ */
+function tooltipFactory ($html) {
+    const tooltipListener = new TooltipListener($html, tippy, hideAll);
+
+    tooltipListener.init();
```

Comment:
let's remove this init here as we'll do this where we init everything else. We want to keep our factories declarative and side-effect free.

we want to return the `tooltipListener` instance from this function